python-lib/dku_plugin_config_loading.py

```diff
@@ -0,0 +1,164 @@
+from dataiku.customrecipe import *
+import dataiku
+import pandas as pd
+from dataiku import pandasutils as pdu
+from dataiku.customrecipe import get_recipe_config
+from language_dict import SUPPORTED_LANGUAGES_SPACY
+from dku_config import DkuConfig
+
+#All plugin parameters name
+MATCHING_PARAMS       = ['case_sensitivity',
+                         'lemmatization',
+                         'unicode_normalization'
+                        ]
+
+def get_input_datasets(role):
+    return dataiku.Dataset(get_input_names_for_role(role)[0])
+
+def load_settings():
```

Comment:
This function is really long - can we make a DkuPluginConfigLoader class with different methods?

Reply:
#ClassyClass ff4722b